addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/core/LxControlJalousie.java

```diff
@@ -0,0 +1,237 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.loxone.core;
+
+import java.io.IOException;
+
+import org.openhab.binding.loxone.core.LxJsonApp3.LxJsonControl;
+
+/**
+ * A jalousie type of control on Loxone Miniserver.
+ * <p>
+ * According to Loxone API documentation, a jalousie control covers:
+ * <ul>
+ * <li>Blinds
+ * <li>Automatic blinds
+ * <li>Automatic blinds integrated
+ * </ul>
+ *
+ * @author Pawel Pieczul - initial commit
+ *
+ */
+public class LxControlJalousie extends LxControl implements LxControlStateListener {
+    /**
+     * A name by which Miniserver refers to jalousie controls
+     */
+    private static final String TYPE_NAME = "jalousie";
+
+    /**
+     * Jalousie is moving up
+     */
+    private static final String STATE_UP = "up";
+    /**
+     * Jalousie is moving down
+     */
+    private static final String STATE_DOWN = "down";
+    /**
+     * The position of the Jalousie, a number from 0 to 1
+     * Jalousie upper position = 0
+     * Jalousie lower position = 1
+     */
+    private static final String STATE_POSITION = "position";
+    /**
+     * The shade position of the Jalousie (blinds), a number from 0 to 1
+     * Blinds are not shaded = 0
+     * Blinds are shaded = 1
+     */
+    @SuppressWarnings("unused")
```

Comment:
This represents a full API in the Miniserver. These features of the API are not used at the moment by the binding. The idea was that keeping them in the file makes it a complete API definition and in case they become needed by the binding in the future, there is no need to return to the documentation.